mrack.spec

```diff
@@ -1,5 +1,5 @@
 # Created by pyp2rpm-3.3.5
-%global srcname mrack
```

Comment:
typo: `pyton3`

Reply:
And actually, python applications doesn't have to start with `python3`. 
As https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_application_naming says:

> Packages that primarily provide applications, services or any kind of executables SHOULD be named according to the general [Fedora naming guidelines](https://docs.fedoraproject.org/en-US/packaging-guidelines/Naming/) (e.g. ansible).

Probably the most correct thing would be to have packages:

- mrack (packaging mrack CLI)
- python3-mrack (core of the mrack - the library to be imported e.g. in tmt)
- python3-mrack-openstack (the provider)

Reply:
I have re-pushed without the patch adding the python- prefix.
We should decide how to divide the package itself i have pushed this PR to initiate discussion basically as we need to agree on something which makes sense for us as well.

My question is, should we create package per each provider from (AWS, Virt, Beaker, Openstack, Podman) and keep static as a base for the library or should we make 

```
└─ mrack   
       └─  python3-mrack-all (maybe without this part)
                     └─ python3-mrack-aws
                     │          └─  python3-mrack (Static)  
                     ├─ python3-mrack-beaker
                     │          └─  python3-mrack (Static)  
                     ├─ python3-mrack-openstack
                     │          └─  python3-mrack (Static)  
                     ├─ python3-mrack-podman
                     │          └─  python3-mrack (Static)  
                     └─ python3-mrack-virt
                                └─  python3-mrack (Static)  

```

Reply:
How would you divide the package?